gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,687 @@
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon start_time
+    path: start_time
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon tcp_flag_syn
+    path: tcp_flag_syn
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon tcp_flag_synack
+    path: tcp_flag_synack
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon tcp_flags
+    path: tcp_flags
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon tcp_retransmission_bytes
+    path: tcp_retransmission_bytes
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon tcp_rtt
+    path: tcp_rtt
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon tcp_rtt_app
+    path: tcp_rtt_app
+    source: log
+    type: string
+pipeline:
+  type: pipeline
+  name: gigamon
+  enabled: true
+  filter:
+    query: "source:gigamon"
+  processors:
+    - type: attribute-remapper
+      name: Map `dns_transaction_id` to `dns.id`
+      enabled: true
+      sources:
+        - dns_transaction_id
+      sourceType: attribute
+      target: dns.id
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `dns_flags` to `dns.flags.rcode`
+      enabled: true
+      sources:
+        - dns_flags
+      sourceType: attribute
+      target: dns.flags.rcode
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map 'http_header_value' to 'http.refer
+      enabled: true
+      sources:
+        - http_header_value
+      sourceType: attribute
+      target: http.referer
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map 'http_request_ts' to 'http.request_id'
+      enabled: true
+      sources:
+        - http_request_ts
+      sourceType: attribute
+      target: http.request_id
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map 'http_host' to 'http.url_details.host'
+      enabled: true
+      sources:
+        - http_host
+      sourceType: attribute
+      target: http.url_details.host
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: user-agent-parser
+      name: Map 'http_user_agent' to 'http.useragent_details.os.family'
+      enabled: true
+      sources:
+        - http_user_agent
+      target: http.useragent_details.os.family
+      encoded: false
+      combineVersionDetails: false
+    - type: grok-parser
+      name: Map 'ts' to 'TimeStamp'
+      enabled: true
+      source: message
```

Comment:
This isn't right, it should be `ts`, the source attribute where we're applying the grok expression. The `message` today has the entire contents. The sample should also be changed to be just the value: `Wed Jul 31 12:05:39 2024`. 

Once done, you should also add a timestamp remapper with the source attribute as whatever you extract out of the grok parser. 

Can you make these changes and test it with your test pipeline? To ensure that the dates you're sending in the logs gets correctly set within Datadog? We drop old logs so please make sure the timestamp you're testing with is within the same day you send it for ingestion.